src/SonarScanner.MSBuild/BootstrapperClass.cs

```diff
@@ -24,16 +24,27 @@
 using System.Linq;
 using System.Reflection;
 using System.Threading.Tasks;
+#if NETCOREAPP2_1
+
+using SonarScanner.MSBuild.AnalysisWarning;
+
+#endif
```

Comment:
In this case, I'd say it's fine to skip the empty lines. It's a continuous block. Otherwise, there should be an empty line before and after to follow the coding style literally.